website/public/index.ejs.html

```diff
@@ -475,6 +496,7 @@ <h2 id="linter-feature-comparison">Features</h2>
               <td class="bad"><span aria-label="no">❌</span></td>
               <td class="bad"><span aria-label="no">❌</span></td>
               <td class="bad"><span aria-label="no">❌</span></td>
+              <td class="bad"><span aria-label="no">❌</span></td>
             </tr>
             <tr>
               <th colspan="2">LSP server</th>
```

Comment:
Blocking: Add missing cell for LSP server.

It looks like their VS Code plugin uses an LSP server, so I'd say ✅ for SonarLint+LSP.